src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/HdcService.kt

```diff
@@ -43,6 +44,12 @@ class HdcService(
     return licenceData.curfewAddress
   }
 
+  @Transactional
+  fun getCurfewTimesByBookingId(bookingId: Long): List<HdcCurfewTimes>? {
+    val licenceData = this.hdcApiClient.getByBookingId(bookingId)
```

Comment:
best to not do two calls to get the same information.. could just return the hdc info as a whole and then pass that through to the eligibility service

Reply:
Now only doing one API call for this as suggested - thank you!